subscription/test/sns_test.py

```diff
@@ -28,63 +28,5 @@ def test_create_topic_client_error(self):
         with self.assertRaises(ClientError):
             self.sns.create_topic("test_topic")
 
-    @patch('json.loads')
```

Comment:
Do all these tests no longer serve any value?

Reply:
No, they don't. I shouldn't have had them in the first place.